RUNBOOK.md

```diff
@@ -0,0 +1,153 @@
+
+### Technical overview
+
+Internal application based on Express.js, using Typescript (compiled to javascript running on node.js) as the main language. Deployed in kubernetes (the HMPPS Cloud Platform) using the configuration found in [helm_deploy](helm_deploy).
+
+### Service Level Agreements (SLAs)
+
+Office hours (Mon-Fri, 09:00-17:00), best efforts.
+
+### Service owner
+
+The `ppud-replacement` team develops and runs this service.
+
+Contact the [#ppud-replacement](https://mojdt.slack.com/archives/C020S8C0K9U) and [#ppud-replacement-devs](https://mojdt.slack.com/archives/C0223AGGQU8) channels on slack.
+
+### Contributing applications, daemons, services, middleware
+
+- Express.js application based on [hmpps-template-typescript](https://github.com/ministryofjustice/hmpps-template-typescript).
+- AWS Elasticache (redis) for caching - configured via [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments).
+
+## System characteristics
+
+### Hours of operation
+
+Available 24/7.
+
+### Infrastructure design
+
+The application runs on the [HMPPS Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/) within the `manage-recalls-<tier>` namespaces (where `tier` is `dev`, `preprod` or `prod`).
+
+**NOTE:** The service runs on the `live` cluster, not the deprecated `live-1` instance.
+
+The main application runs as a deployment named `manage-recalls-ui`. The application is made avialable externally from the cluster via an Ingress.
+
+See the `values-<tier>.yaml` files in the [helm_deploy](helm_deploy) directory for the current configuration of each tier.
+
+### Throttling and partial shutdown
+
+If there is an issue with the service where it is causing load on downstream services and it needs to be shutdown quickly the following command will reduce the number of pod replicas to zero:
+
+```
+kubectl -n manage-recalls-<tier> scale deployment manage-recalls-ui --replicas=0
+```
+
+We do not currently have a strategy in place to throttle requests.
+
+### Environmental differences
+
+Infrastructure wise, all three tiers are identical, but `prod` has the following differences:
+
+- It will have more pod replicas of the main application deployment.
+- As this is live data, you **must** be SC cleared to work on it or access the data held within it.
+
+### Tools
+
+See the [scripts](scripts) directory and the documentation held in [docs](docs) for more details.
+
+## Security and access control
+
+In order to gain access to the `manage-recalls-<tier>` namespaces in kubernetes you will need to be a member of the [ministryofjustice](https://github.com/orgs/ministryofjustice) github organisation and a member of the [ppud-replacement-devs](https://github.com/orgs/ministryofjustice/teams/ppud-replacement-devs) (github) team. Once joined, you should have access to the cluster within 24 hours.
```

Comment:
For anything that takes time to get eg access, is it worth moving to a 'support prerequisites' section so someone who's going to be on a support rota has a clear list of things they need before they join the rota?

Reply:
It's a standard template this is working off - I'd rather not fiddle with the order too much...

Reply:
who provides the template? Happy to. make the point to them, it's something I've seen trip up support rotas before.

Reply:
You know, I can't remember where I got it from now 🤦 .  F' it, I'll move it - will just need to rejig things so it makes sense first.